### ISSUE TITLE
Text selectin issue

### ISSUE THREAD

**Issue by kapilsharma** [coolwanglu/pdf2htmlEX#547]
# Issue

When line spacing between lines is more, selecting text on generated output is frustrating.

Between lines,on generated HTML, if there is space and while selecting, if mouse move to that space, selection becomes `start point` to `top of page`.

# Reproducing
PDF I used to generate HTML can be downloaded from:
http://kapilsharma.github.io/test/pdf2html-issue-sample/pdf2html-issue-sample.pdf
A simple word file created with double spacing and exported to pdf.

Generated HTML is available at:
http://kapilsharma.github.io/test/pdf2html-issue-sample/pdf2html-issue-sample.html
Generated HTML output. Sample paragraph given to test.

I'm stuck at it. Is there any quick solution I can apply?

**Comment by duanyao** [comment]
I don't know why this is happening. However this problem only happens when the cursor is between two lines, not when the cursor is on a line, so you can still select text of multiple lines.

**Comment by kapilsharma** [comment]
Yes, problem is happening when cursor is between two lines. It do not cause much problem (Still not good) on desktop but on tablets/mobile device, it is frustrating where we do not have sharp mouse pointer but fat finger.

**Comment by duanyao** [comment]
It seems this test file doesn't suffer from this issue:
https://github.com/coolwanglu/pdf2htmlEX/tree/master/test/browser_tests/geneve_1564

However I had a hard time to figure out the difference. Do you have any idea?

**Comment by kapilsharma** [comment]
Thanks for reply @duanyao 

Problem occurs when pointer goes in empty space. However but even in test file you mentioned, it can be reproduced as follow:

**Test case**
- On note on left most of file, click and drag from starting 
```
Ce premier cha-
pitre eﬅ fort diﬃ-
cile : & por cette
caſe, il
```
- While mouse button still still down (selecting text), take mouse pointer between third and fourth line, below `&`, after `fe,` and before `il`.
- You will find issue reproduced and selection is not what we expected.

However real test case should be, if there is some space between text (double line spacing is most crucial test case), and while selecting if mouse reach in that space, selection distort.

**This is crucial for mobile devices**, where we do not have sharp mouse pointer and issue occur frequently.

**Comment by kapilsharma** [comment]
Test case on online version http://coolwanglu.github.io/pdf2htmlEX/demo/geneve.html

- On note on right hand side of file, start selecting first para

```
porqoy les He-
briex cõmencent
le ior natrel le
ſoir apres le ſoleil
couchant.
g
Ce mot
```

- While mouse button still down (selecting text), take mouse pointer to end of para in white space after `couchant.`
- You will find issue reproduced and selection is not what we expected.

**Comment by kapilsharma** [comment]
Please note: Issue is only in Firefox.
We tried this in Chrome and IE but issue not reproduced there.
